### PR TITLE
Landing: final polishing

### PR DESCRIPTION
**Pull request by Diogomartf**
In this PR we want to fix, improve and review:
- copy
- spacing
- font sizes
- bugs

### CODE REVIEW

**Comment by berteotti** [comment]
Should we make the navbar sticky?

**Comment by Diogomartf** [comment]
> Should we make the navbar sticky?

We can give that a try and see how we feel

File: packages/landing/components/navbar/Navbar.tsx [hunk 1]
```diff
@@ -5,33 +5,55 @@ import MobileMenu from "./MobileMenu";
...
 
 export function Navbar() {
+  const buttonRef = useRef(null);
+  const [scrollY, setScrollY] = useState(0);
+  const [viewportHeight, setViewportHeight] = useState(600);
+  console.log("scrollY:", scrollY);
+
+  useEffect(() => {
+    if (window.innerHeight) setViewportHeight(window.innerHeight);
+  }, []);
```
Comment (berteotti) [inline packages/landing/components/navbar/Navbar.tsx:18]: Shouldn't the "active button height" be fixed? Maybe we should show it after the user scrolls just a bit (100 pixels)?

File: packages/landing/app/page.tsx [hunk 2]
```diff
@@ -14,36 +14,37 @@ import { QAndAAccordion, TryStacklyBanner } from "@/components";
...
+          href={STACKLY_APP_URL}
+          size="lg"
+          width="fit"
+          className="!py-4 mx-auto text-lg !px-16 md:!px-28 mt-8"
+        >
+          Stack now
+        </ButtonLink>
+        <div className="relative max-w-4xl mx-auto mt-12 mb-24 md:my-20">
+          <Link passHref href={STACKLY_APP_URL}>
+            {/* <div className="absolute w-0.5 h-4 bg-gray-600 bottom-[32px] left-2.5 animate-pulse"></div> */}
```
Comment (berteotti) [inline packages/landing/app/page.tsx:36]:
```suggestion
```

Comment (Diogomartf) [reply packages/landing/app/page.tsx]: It wasn't suppose to commit that. Fixed!

File: packages/landing/ui/text/HeadingText.tsx [hunk 3]
```diff
@@ -6,10 +6,10 @@ const headingTextStyles = cva([], {
...
       6: ["text-[64px] leading-[72px]"],
       5: ["text-[52px] leading-[64px]"],
-      4: ["text-[40px] leading-[56px]"],
-      3: ["text-[32px] leading-[40px]"],
-      2: ["text-[26px] leading-[40px]"],
-      1: ["text-[20px] leading-[32px]"],
+      4: ["text-[26px] leading-[40px] md:text-[40px] md:leading-[56px]"],
+      3: ["text-xl leading-[32px] md:text-[32px] md:leading-[40px]"],
+      2: ["text-[20px] leading-8 md:text-[26px] md:leading-[40px]"],
+      1: ["text-[18px] md:text-[20px] md:leading-[32px]"],
```
Comment (berteotti) [inline packages/landing/ui/text/HeadingText.tsx:12]: Are we going to sync this with app/ui ? Also applies to other /ui update on landing.

File: packages/landing/app/twitter-image.jpg
Comment (berteotti) [inline packages/landing/app/twitter-image.jpg:1]: New twitter image is missing.

Comment (Diogomartf) [reply packages/landing/app/twitter-image.jpg:1]: fixed

File: packages/landing/ui/text/HeadingText.tsx (hunk 3 above)
Comment (Diogomartf) [reply packages/landing/ui/text/HeadingText.tsx:12]: We'll have to do a proper strategy to have responsive fonts on our UI.

As we'll use that UI components on new Swapr liquidity as well. We need to find a solution that will fit all of the different apps.

**Review by berteotti — APPROVED** [review]

**Pull request merged**